### PR TITLE
Fixes #10609 but keeps diona burn mod the same

### LINKED ISSUE #10609: Plasmamen seem to have 75% innate burn resist rather than the intended 50%
Plasmamen have 75% base laser damage resistance, this is applied at all times, even when naked. For reference this amount of resistance makes a naked plasmaman take less burn damage from lasers than humans wearing death squad, nuke ops or ERT armor.
A naked human takes 4 lasers to go into crit, a naked plasmaman (or wearing an assistant suit) takes 14.
Plasmamen security personnel, HoPs and captains get a further 35 value on laser armor value of their respective suits, making them able to tank over 20 lasers before going down. For reference, this makes them more resilient to lasers than nuke ops, death squaddies and security ERT members (which they still outlast naked).
If I am to bring a more extreme case, a HoS plasmaman wearing one riot shield outlasts a Durand in defense mode and using a ranged weaponry armor booster module.

Now, this is where I ask: is this balanced?
To me it feels that it is not, even taking in account how most firefights are based around tases and how there's plenty of non-laser damage to go around this resistance to lasers is extreme and unjustified. Players who grab lasers are often taking advantage of the projectile speed and how they go through windows and glass airlocks and doors, making countering with tasers impossible in common situations involving and bullets easy to dodge because of the distance. To top this off there's pain damage from burns to be considered, a human hit by just two lasers beings to moves slowly due to it, becoming an easy target, a plasmaman needs to take 7 shots to begin suffering from slowdown so they have a ridiculous breather to run away.

Does having to live with the suit in most situations make up for this? Does having to breathe plasma and plasma tanks not being common to replace make up for this? Is not being human enough of a disadvantage?
With my personal experiences I don't think any of those are big enough cons to make up for that but that's where I want people here ponder about it.

If I were to give a suggestion I'd say that removing this reduction completely from them and adding a 10-20 laser defense value on their base suit would be enough to give them some edge against lasers without being near broken as it is. After all they already got an edge against brute by virtue of not bleeding.

Some webms showing their resistance:
Comparison between a naked plasmamen (on lepo and dex+ to negate damage from fire and a lack of plasma), a human and a diona:
http://i.4cdn.org/vg/1466192764617.webm
Naked plasmaman compared to armored (but shieldless) death squad, nuke op and security ERT (commander ERT suits are weaker what the hell).
http://i.4cdn.org/vg/1466196234044.webm
A HoS plasmaman with a riot shield compared to a Durand which I mentioned before:
http://i.4cdn.org/vg/1466198381055.webm
I still need to upload these somewhere else.

### PR DESCRIPTION
**Pull request by Intigracy**
Has the bugfix from #10650
fixes #10609

Diona will be taking half the burn damage that they currently take (which, even though is a result of a bug, a lot of people have seen as reasonable due to their basically complete immunity to other forms of damage)

This keeps the damage at the same values once the bug is fixed.

### CODE REVIEW

**Comment by ComicIronic** [comment]
Actually, 2.5^2 is 6.25, so this is a slight buff. Also, make this PR onto Celt's branch, it doesn't make any sense to keep it separate.

**Comment by Shadowmech88** [comment]
Aren't dionae put into crit by literally one laser shot right now?
I get that they need a weakness, but their current weakness is a bit high _and_ it's unintentional.

**Comment by Intigracy** [comment]
And their strength is being essentially invulnerable to every other damage
type

**Comment by Intigracy** [comment]
idk why it's need changelog it's not changing at all

though the brute modifiers and other species ones are so nvm I'll changelog it later since somehow I inherited this change

**Comment by Shadowmech88** [comment]
> idk why it's need changelog it's not changing at all

The burn damage taken by plasmamen is changing from 75% to 50%, isn't it?

**Comment by Intigracy** [comment]
```
+author: DrCelt / Intigracy
 +delete-after: True
 +changes: 
 +- bugfix: "(DrCelt) As an effect of a bugfix, all species are now a little less weak/strong against damage types that are modified for them. Think plasmamen burn damage resistance."
 +- tweak: "(Intigracy) Dionae burn damage increase is an exception, it remains unchanged." 
```

**Comment by clusterfack** [comment]
Removes the comments they add nothing to the code :^)

**Comment by Intigracy** [comment]
celt a shit

File: html/changelogs/DrInti.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author: DrCelt / Intigracy
+delete-after: True
+changes: 
+- bugfix: "(DrCelt) As an effect of a bugfix, all species are now a little less weak/strong against damage types that are modified for them. Think plasmamen burn damage resistance."
```
Comment (ghost) [inline html/changelogs/DrInti.yml]: that's a stupid way to word that.
Fixed damage species modifiers not applying itself twice.

Comment (Intigracy) [inline html/changelogs/DrInti.yml]: My way at least explains what that means.

Comment (ghost) [inline html/changelogs/DrInti.yml]: @Intigracy Then elaborate in a different line. This doesn't even say what the bug fix is.

File: code/modules/mob/living/carbon/species.dm [hunk 2]
```diff
@@ -714,7 +714,7 @@ var/global/list/whitelisted_species = list("Human")
 	flesh_color = "#907E4A"
 
 	has_mutant_race = 0
-	burn_mod = 2.5 //treeeeees
+	burn_mod = 6.5 //treeeeees
```
Comment (ghost) [inline code/modules/mob/living/carbon/species.dm]: 2.5^2 = 6.25, not 6.5. This doesn't keep diona burn damage the same

File: code/modules/mob/living/carbon/species.dm [hunk 3]
```diff
@@ -714,7 +714,7 @@ var/global/list/whitelisted_species = list("Human")
 	flesh_color = "#907E4A"
 
 	has_mutant_race = 0
-	burn_mod = 2.5 //treeeeees
+	burn_mod = 6.2 //treeeeees
```
Comment (ghost) [inline code/modules/mob/living/carbon/species.dm:717]: 6.25 isn't the same as 6.2, @Intigracy

Comment (Intigracy) [inline code/modules/mob/living/carbon/species.dm:717]: idc

File: html/changelogs/DrInti.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+author: DrCelt / Intigracy
+delete-after: True
+changes: 
+- bugfix: "(DrCelt) Fixed a bug where species damage modifiers were applied twice."
+- tweak: "(DrCelt) As a result of this, brute/burn modifiers are slightly less effective when it comes to resistance/weakness (less damage resisted, less damage taken, respectively)"
+- tweak: "(Intigracy) Dionae burn damage is an exception, it remains unchanged."
```
Comment (ghost) [inline html/changelogs/DrInti.yml]: change this to "has decreased by 0.05" then

**Comment by Intigracy** [comment]
>didn't delete my branch




cluuuuusterrrrrrfaaaaackkkkk

**Comment by Kammerjunk** [comment]
So dionae dying to one shot is intended?
Some poor guy kinda took 265 burn damage when normal people would have taken 40 (and normal lasers, which do 30, would still do 198.75)

**Pull request merged**